WeakAuras/Profiling.lua

```diff
@@ -6,6 +6,8 @@ local profileData = {}
 profileData.systems = {}
 profileData.auras = {}
 
+WeakAuras.profileData = profileData
```

Comment:
This is unrelated to the rest of the pull request. Did you want to split it out and submit it separately?

Reply:
sure i can, there are also a few other UI related stuff that can be splited, but the border changes are a bit too intricate with anchor changes to be splited